README.adoc

```diff
@@ -22,7 +22,7 @@ Mutual TLS means that connecting clients need to have a client certificate befor
 |Client certificates will expire, and hence need to recreate these for all clients periodically, which introduces a lot of overhead and potential for errors
 
 |
-|Have to maintain a CA for signing client certs
+|Have to maintain a CA for signing client certs. (Have you thought about how you will revoke access should this need arise?)
```

Comment:
When I say self-signed, I suppose what I'm getting at is that I'm pretty much side-stepping all the complexity that comes with running a CA.

Also, for internal stuff there /might/ already be better solutions out there using cfssl which is what is usually used with k8 I believe. I'd defer to the devops guys on that one.